aten/src/ATen/cuda/CUDABlas.h

```diff
@@ -34,6 +34,10 @@ void gemm<double>(CUDABLAS_GEMM_ARGTYPES(double));
 template <>
 void gemm<float>(CUDABLAS_GEMM_ARGTYPES(float));
 template <>
+void gemv<c10::complex<double>>(CUDABLAS_GEMV_ARGTYPES(c10::complex<double>));
```

Comment:
I think you need to move this down to `gemv`, you are currently putting it to `gemm`.

Reply:
oh yeah my bad. updated